django/thunderstore/account/api/experimental/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from thunderstore.account.api.experimental.views import TokenExperimentalApiView
+
+urls = [
+    path(
+        "auth/token/",
```

Comment:
maybe we could call this `session-token` instead, as it's to be created from the session and bound to it?